src/js/simpleSmtpServer.js

```diff
@@ -0,0 +1,84 @@
+// A module to wire in simplesmtp to handle incoming messages.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp.simpleSmtpServer");
+
+var simplesmtp = require("simplesmtp");
+var fs         = require("fs");
+
+gpii.test.mail.smtp.simpleSmtpServer.handleStartData = function (that, connection) {
+    var timestamp = (new Date()).getTime();
+    that.options.messageFile = that.options.config.outputDir + "/message-" + timestamp + ".txt";
```

Comment:
Someone else mentioned "members" for this type of thing, but you've continued to push for top-levelling things like this.  I'd like to know the history of the "members" pattern if you know it.